tests/spec/utils-spec.js

```diff
@@ -0,0 +1,41 @@
+describe('F2 utility methods', function() {
+
+	beforeEachReloadF2();
+
+	it('F2.version should return a version number (SemVer as String)', function() {
+		//https://github.com/sindresorhus/semver-regex
+		var testVersion = /\bv?(?:0|[1-9][0-9]*)\.(?:0|[1-9][0-9]*)\.(?:0|[1-9][0-9]*)(?:-[\da-z\-]+(?:\.[\da-z\-]+)*)?(?:\+[\da-z\-]+(?:\.[\da-z\-]+)*)?\b/ig.test(F2.version());
```

Comment:
`semver` package used by npm would be saner but something something ain't broke don't fix https://www.npmjs.com/package/semver